### PR TITLE
added occupied habitat - stats and image export

### PR DESCRIPTION
**Pull request by thanrobinson**

### CODE REVIEW

**Comment by thanrobinson** [comment]
[add occupied habitat to scl_classification and ensure property persist through stats](https://trello.com/c/ZqZL3O2B/503-add-occupied-habitat-to-sclclassification-and-ensure-property-persist-through-stats)

File: src/task.py [hunk 1]
```diff
@@ -540,10 +567,33 @@ def is_gridcell_unique(self, df):
         return df_counts.index.is_unique
 
     def calc(self):
+        print(self.df_adhoc)
```
Comment (saanobhaai) [inline src/task.py:570]: We should comment this so a future developer knows why we have to print and not remove them. And add a card to refactor so we don't have to do this.

File: src/task.py [hunk 2]
```diff
@@ -540,10 +567,33 @@ def is_gridcell_unique(self, df):
...
+            .clipToCollection(observed_grid)
+            .rename(OCCUPIED_HABITAT_AREA)
+        )
+        self.export_image_ee(occupied_habitat, "pothab/occupied_habitat")
+
+        area = ee.Image.pixelArea().divide(1000000).updateMask(self.watermask)
+        str_hab_area = area.updateMask(self.structural_habitat).rename(
+            STRUCTURAL_HABITAT_AREA
+        )
+        habitat_area = occupied_habitat.addBands(str_hab_area)
```
Comment (saanobhaai) [inline src/task.py:591]: How about set this to `self.habitat_area` (and initialize in `init`), and then remove the additional `habitat_area_image` arg in `attribute_areas`?

File: src/task.py [hunk 3]
```diff
@@ -252,13 +271,14 @@ def _max_frequency(feat):
...
         # ]
-
-        if savefc:
-            dfexport = self._prep_obs_df(df)
-            if not dfexport.empty:
-                obs_export = self.df2fc(dfexport)
+        obs_export = None
+        dfexport = self._prep_obs_df(df)
+        if not dfexport.empty:
+            print(dfexport)
```
Comment (saanobhaai) [inline src/task.py:277]: remove

**Review by saanobhaai — CHANGES_REQUESTED** [review]

**Pull request merged**